lib/middleware.js

```diff
@@ -50,6 +50,13 @@ class Kevin {
                 requestPath.replace(/^\//, "").replace(/\.js$/, ""),
             // Only build assets; don't handle serving them. This is useful if you want to do
             // something with the built asset before serving it.
+            selectConfigNameGivenReqPath = (reqPath, configNames) => {
+                if (!configNames) {
+                    return null;
+                }
+
+                return configNames[0];
```

Comment:
would it make sense to have a check for whether `configNames` has a length that is greater than 1, and if so, log a warning that there are multiple configs for the entrypoint and we're just choosing the first one found?

Reply:
I tossed in a logError line - i don't pass the assetName to the selectConfigName invocation and i don't know that I should, as it may be exposing too much internals of how kevin works. Do you think that's good enough?

Reply:
Maybe including the request path and config names?
```js
logError(`Multiple configNames found for ${reqPath}: ${configNames.join(",")}. Using first one.`);
```

Reply:
done